tests/components/grid.test.ts

```diff
@@ -1341,16 +1315,7 @@ describe("Events on Grid update viewport correctly", () => {
     expect(model.getters.getSelectedZone()).toEqual(toZone("G1"));
     expect(model.getters.getActiveMainViewport().left).toEqual(6);
     expect(model.getters.getActiveSheetScrollInfo().scrollX).toEqual(3 * DEFAULT_CELL_WIDTH);
-    document.activeElement!.dispatchEvent(
-      // scroll completely to the right
-      new WheelEvent("wheel", {
-        deltaX: 0,
-        deltaY: -4 * DEFAULT_CELL_WIDTH,
-        shiftKey: true,
-        deltaMode: 0,
-        bubbles: true,
-      })
-    );
+    triggerWheelEvent(".o-grid input", { deltaY: -4 * DEFAULT_CELL_WIDTH, shiftKey: true });
```

Comment:
`triggerWheelEvent(document.activeElement!, ...`
to keep the same test as before

Reply:
@rrahir wanted to specify the targets to make tests clearer ... https://github.com/odoo/o-spreadsheet/pull/2080#discussion_r1108561053